script.js

```diff
@@ -0,0 +1,29 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const allShapes = ['rock','paper','scissors'];
+
+function computerPlay() {   
+    let randomValue = Math.floor(Math.random()*allShapes.length);
+    return allShapes[randomValue];  
+}
+
+function humanPlay() {
+    return prompt("Enter value:");  
+}
+
+function roundPlay(allShapes, humanVal) {
+    if (allShapes == humanVal) {
+        return("draw");
```

Comment:
this makes no sense, allShapes == humanVal is draw?

Reply:
also same problem with allShapes below, please test everything before committing